onedrive.1.in

```diff
@@ -48,6 +48,9 @@ Display the sync status of the client \- no sync will be performed.
 \fB\-d \-\-download\-only\fP
 Only download remote changes
 .TP
+\fB\-d \-\-dry\-run\fP
+Perform a trial sync with no changes made. Can ONLY be used with --synchronize. Will be ignored for --monitor
+.TP
```

Comment:
The `-d` is used two times. Maybe use `-n` which is very common alias for `--dry-run`?